source/tutorial/ex1.rst

```diff
@@ -0,0 +1,42 @@
+.. _Exercise 1:
+
+Exercise 1 \\- \ Build Bootloader for Qemu
+---------------
+
+.. note::
+  **In this exercise, we'll learn how to build slimbootloader for QEMU emulator.**
+
+
+You can build |SPN| with the following steps:
+
+* Launch Command Prompt
+* Go to SlimBootLoader source code folder by type-in cmd::
```

Comment:
I think "by type-in cmd" is slightly confusing. Can we use "using the following command".

Reply:
good comment, fixed